### DISCUSSION TITLE
[feature idea + feedback] Copilot Stats

### DISCUSSION THREAD

**Discussion by abhishek7h** [community/community#10733] [Copilot Conversations]
I have an idea, there should be a Copilot menu where we can see our Copilot stats, for example how many line it has auto-completed for us, and like how much time we spent programming that day. This feature would be pretty cool.

Feedback:
Copilot has been working great for me and I like it :D.

**Comment by Moulick** [comment]
This would be super useful to justify to managers how much productivity Copilot is increasing and getting businesses to buy licenses.
